eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/ZmqWatcher.scala

```diff
@@ -251,6 +247,50 @@ class ZmqWatcher(chainHash: ByteVector32, blockCount: AtomicLong, client: Extend
     }
   }
 
+  // TODO: move to a separate actor that listens to CurrentBlockCount and manages utxos for RBF
+  def checkUtxos(): Future[Unit] = {
+    case class Utxo(txId: ByteVector32, amount: MilliBtc, confirmations: Long, safe: Boolean)
+
+    def listUnspent(): Future[Seq[Utxo]] = client.rpcClient.invoke("listunspent", /* minconf */ 0).collect {
```

Comment:
Is the commented `minconf` normal?

Reply:
Yes, it's to indicate the name of the RPC parameter, otherwise there's no way to document that inside the code (since bitcoin requests are not typed).